plugins/opensource/fullscreen.md

```diff
@@ -27,6 +27,7 @@ tinymce.init({
   toolbar: 'fullscreen'
 });
 ```
+> **NOTE**: As of {{site.productname}} 5.10.6, toggling fullscreen mode with the `fullscreen` plugin also fires the `ResizeEditor` event.
```

Comment:
```suggestion
```
This should not be here because it's expected it should since the editor resizes. Additionally it's inconsistent with everything else, as nothing else we have mentions what it triggers so this only leads to confusion/misunderstanding.

Reply:
> it’s expected it should fire since the editor resizes

Which argues for the language used in the _Toggling fullscreen mode with the `fullscreen` plugin now also fires the `ResizeEditor` event_ note to be changed.

It currently reads:

> As part of that fix, a change was made to the `fullscreen` plugin.

It should, perhaps, and given ‘expected’ above, read:

> As part of that fix, an error in the `fullscreen` plugin was corrected.

Also, this change constitutes something of an argument for retaining the NOTE above, albeit also with an edit to emphasise the correction:

> **NOTE**: {{site.productname}} 5.10.6 includes a fix to the `fullscreen` plugin. When toggling fullscreen mode with the `fullscreen` plugin it now fires the `ResizeEditor` event, as expected.

Reply:
To quote my commit message:

> Moved documentation of `ResizeEditor` event firing when `fullscreen` plugin toggles the fullscreen mode.
> 
> It was a NOTE admonition in`fullscreen.md`.
> 
> It is now part of the description field for the `ResizeEditor` event in the *Editor core events* table of `events.md`. (And is no longer an admonition.)